augly/video/augmenters/ffmpeg/stack.py

```diff
@@ -26,30 +24,40 @@ def __init__(
         self.use_second_audio = use_second_audio
         self.orientation = orientation
 
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+    def get_command(
+        self, video_path: str, output_path: Optional[str] = None
+    ) -> List[str]:
         """
         Stacks two videos together
 
-        @param in_stream: the FFMPEG object of the video
+        @param video_path: the path to the video to be augmented
 
-        @returns: a tuple containing the FFMPEG object with the augmentation
-            applied and a dictionary with any output arguments as necessary
+        @param output_path: the path in which the resulting video will be stored.
+            If not passed in, the original video file will be overwritten
+
+        @returns: a list of strings containing the CLI FFMPEG command for
+            the augmentation
         """
-        video_info = get_video_info(kwargs["video_path"])
-
-        second_video = ffmpeg.input(self.second_video_path)
-        scaled_second_video = second_video.filter(
-            "scale", **{"width": video_info["width"], "height": video_info["height"]}
-        )
-        stack_video = ffmpeg.filter([in_stream, scaled_second_video], self.orientation)
-
-        return (
-            (
-                ffmpeg.concat(stack_video, second_video.audio, v=1, a=1)
-                if self.use_second_audio
-                else stack_video
-            ),
-            {},
-        )
+        video_info = get_video_info(video_path)
+
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-i",
+            self.second_video_path,
+            "-filter_complex",
+            f"[1:v]scale={video_info['width']}:{video_info['height']}[1v];"
```

Comment:
Sanity check: with all these ffmpeg command changes you've made, you have checked that the output videos look the same for all these augmentations you're changing, correct? (it's ok if the files are not identical because the compression rates are different, but they should *look* roughly the same)

Reply:
Yes, I have a script where I run the command with VidGear and compare it to the output of AugLy